src/Umbraco.Core/IO/IOHelper.cs

```diff
@@ -13,7 +13,9 @@
 namespace Umbraco.Core.IO
 {
 	public static class IOHelper
-    {
+	{
+	    public static bool IAmUnitTestingSoNeverUseHttpContextEver = false;

```

Comment:
:)
HttpContext.Current isn't null, but you can't use Server.MapPath and HostingEnvironment.MapPath when unit testing.